delivery/services/organise_service.py

```diff
@@ -116,6 +115,7 @@ def organise_project(
                     organised_project_runfolder_path,
                     lanes))
         # symlink the project files
+        
```

Comment:
No line should be added here, the comment above should belong to the same block as the code that follows, see https://peps.python.org/pep-0008/#block-comments